contracts/clients/src/signer/defaultDomain.ts

```diff
@@ -1,4 +1,3 @@
-import { arrayify } from "@ethersproject/bytes";
-import { keccak256 } from "@ethersproject/keccak256";
```

Comment:
We may end up wanting to move these back to more specific imports depending on what we decide in https://github.com/jzaki/bls-wallet/issues/109